### PR TITLE
Profiles refactor

### PR DESCRIPTION
**Pull request by recalcitrantsupplant**
All endpoints have the new profiles refactor implemented.

Test results:
vocprez only enabled: all pass
spaceprez only enabled: all pass
vocprez and spaceprez both enabled: 3 tests for the system home alt mediatypes fail. The alt profiles are not included in this page as a div. Do we want alt profiles /mediatypes for the system home itself? It is implemented for vocprez and spaceprez *specific* homepages,  but not the additional "system home" which is added when >1 prez is enabled. I think we need to decide system behaviour before implementing or fixing anything

### CODE REVIEW

File: prez/profiles/spaceprez_default_profiles.ttl [hunk 1]
```diff
@@ -85,11 +85,11 @@ altr-ext:hasResourceFormat owl:propertyChainAxiom ( dcterms:conformsTo dcterms:f
     altr-ext:constrainsClass
         geo:FeatureCollection ,
         geo:Feature ,
-        prez:DatasetList ,
         prez:FeatureCollectionList ,
         prez:FeatureList ,
         prez:Conformance ,
-        prez:HomePage ;
+        prez:SpacePrezHome ,
+        prez:VocPrezHome ;
```
Comment (nicholascar) [inline prez/profiles/spaceprez_default_profiles.ttl:92]: Let's leave this here for now but later shift to the VP file

File: prez/templates/mem.html [hunk 2]
```diff
@@ -4,11 +4,11 @@
 <div id="content-col-main">
     <h1>{{ label }}</h1>
     <p>{{ comment }}</p>
-    <p>Instance URI: <a href="{{ uri }}">{{ uri }}</a></p>
+{#    <p>Instance URI: <a href="{{ uri }}">{{ uri }}</a></p>#}
```
Comment (nicholascar) [inline prez/templates/mem.html:7]: Don't be scared: just remove things!

File: prez/test_profiles.py [hunk 3]
```diff
@@ -0,0 +1,95 @@
+# tests profiles/generate_profiles
```
Comment (nicholascar) [inline prez/test_profiles.py:1]: Did I originally make this and just not complete it? Should be a proper pytest file in the end...

**Review by nicholascar — APPROVED** [review]
Some minor comments only:

Fine from my point-of-view as I know what's going on here. May be more spooky for Jamie. Will have to test Jamie's search additions on top of these changes.

File: prez/test_profiles.py (hunk 3 above)
Comment (recalcitrantsupplant) [reply prez/test_profiles.py:1]: looks like your original stub

File: prez/templates/mem.html (hunk 2 above)
Comment (recalcitrantsupplant) [reply prez/templates/mem.html:7]: will do

File: prez/profiles/spaceprez_default_profiles.ttl (hunk 1 above)
Comment (recalcitrantsupplant) [reply prez/profiles/spaceprez_default_profiles.ttl:92]: Yes we can do a review of all the 'standard' profiles at some point

**Pull request merged**